app/model/schema/token.py

```diff
@@ -89,6 +117,54 @@ class SearchTransferHistoryRequest(BaseModel):
         default=None, description="source event of transfer"
     )
     data: Optional[str] = Field(default=None, description="source event data")
+    transaction_hash: Optional[str] = Field(
+        default=None, description="transaction hash"
+    )
+    created_from: Optional[datetime] = Field(
+        default=None, description="created from datetime"
+    )
+    created_to: Optional[datetime] = Field(
+        default=None, description="created to datetime"
+    )
+    value: Optional[int] = Field(default=None, description="value")
+    value_operator: Optional[ValueOperator] = Field(
+        default=ValueOperator.EQUAL,
+        description="value filter condition(0: equal, 1: greater than, 2: less than)",
+    )
+    sort_item: Optional[SearchTransferHistorySortItem] = Field(
+        default=SearchTransferHistorySortItem.id, description="sort item"
+    )
+    sort_order: Optional[SortOrder] = Field(
+        default=SortOrder.ASC, description="sort order"
+    )
+
+
+@dataclass
+class ListAllTransferApprovalHistoryQuery:
+    offset: Annotated[Optional[int], Query(description="start position", ge=0)] = None
+    limit: Annotated[Optional[int], Query(description="number of set", ge=0)] = None
+    value: Annotated[Optional[int], Query(description="value")] = None
+    value_operator: Annotated[
+        Optional[ValueOperator],
+        Query(
+            description="value filter condition(0: equal, 1: greater than, 2: less than)"
+        ),
+    ] = ValueOperator.EQUAL
```

Comment:
Added the parameters of the filters added to the **Search** API that also can be used in GET requests to the **List** API.
(Filters which expect `datetime` type is not added because it cannot be used in GET.)